README.md

```diff
@@ -72,7 +72,13 @@ const Logger = winston.Logger;
 const Console = winston.transports.Console;
 
 // Imports the Google Cloud client library for Winston
-const LoggingWinston = require('@google-cloud/logging-winston');
+
+// Node 6+
+const {LoggingWinston} = require('@google-cloud/logging-winston');
+
+// Node 4+
+// const LoggingWinston = require('@google-cloud/logging-winston').LoggingWinston;
+
```

Comment:
While you're here, add a TS/ESM style important example as well?

Reply:
Done